var/spack/repos/builtin/packages/cpio/package.py

```diff
@@ -13,12 +14,20 @@ class Cpio(AutotoolsPackage, GNUMirrorPackage):
     homepage = "https://www.gnu.org/software/cpio/"
     gnu_mirror_path = "cpio/cpio-2.13.tar.gz"
 
+    executables = ['cpio']
```

Comment:
```suggestion
    executables = ['^cpio$']
```
This will prevent it from matching executables with prefixes/suffixes